mise.toml

```diff
@@ -220,16 +220,20 @@ env._.path = "./server/node_modules/.bin"
 dir = "server"
 run = "email dev -p 3050 --dir src/emails"
 
-[tasks."server:checklist"]
+[tasks."server:checks-small"]
 run = [
   "mise run server:install",
   "mise run server:format",
   "mise run server:lint",
   "mise run server:check",
-  "mise run server:test-medium --run",
   "mise run server:test --run",
 ]
 
+[tasks."server:checks-medium"]
+run = ["mise run server:install", "mise run server:test-medium --run"]
+
+[tasks."server:checks"]
+run = ["mise run server:checks-small", "mise run server:checks-medium"]
```

Comment:
With install in the run block, it will run in parallel with other tasks. I think we need `depends = ["server:install"]` instead?